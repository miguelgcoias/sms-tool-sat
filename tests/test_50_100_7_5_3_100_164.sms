50
86 2 90 2 1 1
0 3 6 3 1 1 1
70 2 72 2 1 1
10 7 31 1 7
14 4 22 1 4
78 5 93 3 1 2 2
45 4 49 3 1 2 1
88 7 102 3 3 1 3
100 5 105 2 2 3
97 7 111 1 7
0 2 4 2 1 1
43 6 61 5 1 1 1 1 2
3 7 24 4 3 1 2 1
58 1 60 1 1
19 4 23 4 1 1 1 1
99 2 103 2 1 1
9 4 17 1 4
34 7 55 1 7
43 2 49 2 1 1
77 7 84 2 3 4
48 2 50 2 1 1
54 7 61 1 7
31 4 43 2 2 2
65 2 69 2 1 1
72 6 84 4 1 2 2 1
67 6 79 4 2 1 1 2
64 6 70 2 2 4
45 5 55 5 1 1 1 1 1
69 6 87 5 1 1 1 1 2
54 7 68 2 4 3
89 5 104 3 2 1 2
22 4 34 4 1 1 1 1
61 2 67 2 1 1
98 5 108 3 3 1 1
65 6 83 4 2 1 2 1
70 2 74 2 1 1
54 5 64 1 5
18 6 24 1 6
24 4 36 4 1 1 1 1
35 3 44 3 1 1 1
59 6 65 3 2 2 2
31 1 33 1 1
89 4 101 3 1 2 1
48 1 51 1 1
2 7 16 3 1 3 3
37 1 39 1 1
42 1 43 1 1
33 2 37 2 1 1
31 1 32 1 1
45 5 60 3 1 3 1
5 41 21 19 24 50
0
3 28 22 32
0
1 4
5 13 36 38 7 30
2 17 11
7 21 49 18 33 38 20 3
8 30 26 18 39 27 8 6 15
6 8 3 16 7 21 31
0
3 11 49 4
1 2
5 17 32 7 45 46
1 11
2 47 40
0
2 46 47
1 49
7 21 19 35 11 39 38 49
3 39 49 15
5 38 44 32 48 45
1 4
5 32 47 48 40 4
4 28 21 50 44
7 44 45 7 19 15 36 21
4 28 45 2 46
2 48 42
6 2 17 41 38 12 24
3 7 39 19
6 41 37 45 50 18 5
0
2 37 48
6 24 46 21 2 31 44
3 23 22 12
5 49 45 38 23 46
2 13 48
1 45
2 38 17
1 17
5 15 46 28 38 19
2 11 45
7 3 11 39 1 24 21 29
3 48 18 7
0
1 39
3 48 5 49
0
0
2 11 38
