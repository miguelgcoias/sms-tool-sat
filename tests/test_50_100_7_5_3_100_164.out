18
2 1 4 5
7 45 46 48
11 0 3
14 58
17 9
19 43 44
28 49 50 51 52 53
30 60 64
32 27 28 29 30
38 18
39 24 25 26 34
40 35 37 39
42 32
45 2 6 13
46 38
48 33 36
49 31
50 54 55 59
